pyqttoolkit/views/shiftable_combo_box.py

```diff
@@ -0,0 +1,68 @@
+# pyqttoolkit
+# Copyright (C) 2018-2019, Simmovation Ltd.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+from PyQt5.Qt import QHBoxLayout, QWidget, pyqtSignal
+from PyQt5.QtWidgets import QToolButton
+#pylint: enable=no-name-in-module
+
+from pyqttoolkit.properties import AutoProperty, bind
+from .combo_box import BindableComboBox
+
+
+class ShiftableComboBox(QWidget):
```

Comment:
Can we change the name to `SteppableComboBox`? `Shiftable` to me sounds more like you can move it about the page